docs/hub/webhooks-guide-metadata-review.md

```diff
@@ -0,0 +1,207 @@
+# Webhook guide: Setup an automatic metadata quality review for models and datasets 
+
+This guide will walk you through creating a system that reacts to changes to a user's or organization's models or datasets on the Hub and creates a 'metadata review' for the changed repository. 
+
+## What are we building and why?
+
+Before we dive into the technical details involved in this particular workflow, we'll quickly outline what we're creating and why. 
+
+[Model cards](https://huggingface.co/docs/hub/model-cards) and [dataset cards](https://huggingface.co/docs/hub/datasets-cards) are essential tools for documenting machine learning models and datasets. The Hub uses a `README.md` file containing a [YAML](https://en.wikipedia.org/wiki/YAML) header block to generate model and dataset cards. This `YAML` section defines metadata relating to the model or dataset. For example: 
+
+```yaml
+---
+language: 
+  - "List of ISO 639-1 code for your language"
+  - lang1
+  - lang2
+tags:
+- tag1
+- tag2
+license: "any valid license identifier"
+datasets:
+- dataset1
+---
+```
+
+This metadata contains essential information about your model or dataset for potential users. The license, for example, defines the terms under which a model or dataset can be used. Hub users can also use the fields defined in the `YAML` metadata as filters for identifying models or datasets that fit specific criteria. 
+
+Since the metadata defined in this block is essential for potential users of our models and datasets, it is important that we complete this section. In a team or organization setting, users pushing models and datasets to the Hub may have differing familiarity with the importance of this YAML metadata block. While someone in a team could take on the responsibility of reviewing this metadata, there may instead be some automation we can do to help us with this problem. The result will be a metadata review report automatically posted or updated when a repository on the Hub changes. 
+
+TODO screenshot of the report.
+
+## Using the Hub Client Library to create a model review card 
+
+The `huggingface_hub` is a Python library that allows you to interact with the Hugging Face Hub. We can use this library to download model and dataset cards from the Hugging Face hub using the `DatasetCard.load` or `ModelCard.load` methods. In particular, we'll use these methods to load a Python Dictionary, which contains the metadata defined in the `YAML` of our model or dataset card. We'll create a small Python function to wrap these methods and do some exception handling. 
+
+```python
+from huggingface_hub import DatasetCard, ModelCard
+from huggingface_hub.utils import EntryNotFoundError 
+
+def load_repo_card(repo_type, repo_name):
```

Comment:
```suggestion
def load_repo_card_metadata(repo_type, repo_name):
```

I'd make explicit the fact that it's returning only the metadata part, not the textual / free-form part